### PR TITLE
Nuget Package generation for Azure DCAP for Windows.

### PR DESCRIPTION
**Pull request by msft-jonnadul**

### CODE REVIEW

File: src/Windows/GeneratePackage/InstallAzureDCAP.ps1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+)
+
+if (-not (Test-Path -Path $localPath))
+{
+    Write-Host "[ERROR]: $localPath does not exist, please provide a valid path"
+}
+else
+{
+    Copy-Item "..\dll\dcap_quoteprov.dll" -Destination $localPath
+    Copy-Item "..\dll\libcurl.dll" -Destination $localPath
```
Comment (LarryOsterman) [inline src/Windows/GeneratePackage/InstallAzureDCAP.ps1]: Why are you copying all these DLLs? None of them should be needed.

File: src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <!-- Dependencies are automatically installed when the package is installed -->
+        <dependencies>
+            <dependency id="Newtonsoft.Json" version="9.0" />
+        </dependencies>
+    </metadata>
+
+    <!-- A readme.txt to display when the package is installed -->
+    <files>
+        <file src="..\dll\x64\Release\dcap_quoteprov.dll" target="dll" />
+        <file src="..\dll\x64\Release\libcurl.dll" target="dll" />
```
Comment (LarryOsterman) [inline src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec]: All you should need is the dcap_quoteprov.dll - none of the others are necessary.

File: src/Windows/GeneratePackage/InstallAzureDCAP.ps1 [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    Copy-Item "..\dll\ssleay32.dll" -Destination $localPath
+    Copy-Item "..\dll\zlib.dll" -Destination $localPath
+    Copy-Item "..\crt\curl-ca-bundle.crt" -Destination $localPath
+    
+    $env:Path += ";" + $localPath
+
+    [Environment]::SetEnvironmentVariable
+     ("Path", $env:Path, [System.EnvironmentVariableTarget]::Machine)
+
+    if ((-not (Test-Path -Path "$env:windir\System32\msvcp140.dll")) -or
```
Comment (LarryOsterman) [inline src/Windows/GeneratePackage/InstallAzureDCAP.ps1]: All of this is unnecessary.

File: src/Windows/GeneratePackage/InstallAzureDCAP.ps1 (hunk 1 above)
Comment (msft-jonnadul) [reply src/Windows/GeneratePackage/InstallAzureDCAP.ps1]: Removed all this.

File: src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec (hunk 2 above)
Comment (msft-jonnadul) [reply src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec]: Removed all this.

File: src/Windows/GeneratePackage/InstallAzureDCAP.ps1 (hunk 3 above)
Comment (msft-jonnadul) [reply src/Windows/GeneratePackage/InstallAzureDCAP.ps1]: Removed all this.

**Pull request merged**